tests/acceptance/new.spec.ts

```diff
@@ -16,12 +16,17 @@ describe('Acceptance: ng new', function () {
 
     spyOn(console, 'error');
 
+    // copy custom collections to node_modules, so we can use with ng new
+    // it is a bit dirty, but bootstrap-local tricks won't work here
+    fs.copySync('./tests/collections/@custom', './node_modules/@custom');
```

Comment:
Would a symbolic link work here? Since it's a `beforeEach`, copying will take a long time.